content/concepts/introduction/users.md

```diff
@@ -0,0 +1,61 @@
+| **Huddle01** | Huddle01 aims to reduce costs by 95% compared to traditional centralized communication platforms like Zoom, Skype, and Google Meet, partly due to its efficient p2p infrastructure powered by libp2p. |
+| **Espresso Systems** | The implementation of libp2p has facilitated Espresso's goal of enabling cross-chain interactions that operate as if on one chain. |
+| **Magi (by a16z)** | Magi is an OP Stack rollup client developed by Andreessen Horowitz (a16z) that implements libp2p as part of its networking stack. |
+| **Berty** | Berty is a secure, privacy-focused messaging app leveraging a custom version of OrbitDB on top of libp2p to organize and store messages in a decentralized manner. |
+| **Flow** | Flow has established itself as a leading blockchain for NFTs and decentralized applications, partly due to its efficient networking layer built on libp2p. |
+| **Oasis** | Oasis Network utilizes libp2p for peer-to-peer communication in its consensus layer, which is responsible for maintaining the network's state and validating transactions. |
+| **Mina** | By leveraging libp2p, Mina has achieved a lightweight blockchain of only 22KB, enabling efficient scalability and decentralized communication. This has significantly enhanced the network's performance and accessibility. |
+| **EdgeVPN** | EdgeVPN uses a lightweight blockchain built on top of libp2p to store and propagate network metadata, such as service UUIDs, IP addresses, and DNS records. This blockchain serves as a coordination mechanism between nodes.|
+| **Status-go / status.im** | Status has developed nim-libp2p, a Nim implementation of libp2p, which is used in projects like Codex, Waku, and Nimbus. |
+| **Fluence Network, nox, aquavm** | Fluence's use of libp2p has contributed to its ability to provide verifiable computation, a crucial feature for ensuring trust in decentralized systems. |
+| **Ceramic** | Ceramic's use of libp2p has contributed to its ability to handle high-volume application data with web-scale volume and latency; libp2p has also enabled IDX, Ceramic's decentralized identity solution. |
+| **Nethermind, juno, dotnet-libp2p** | By leveraging libp2p, Nethermind has positioned its projects (Juno and dotnet-libp2p) to contribute significantly to the decentralization and scalability of blockchain networks, particularly in the Ethereum and Starknet ecosystems. |
+| **Drand / Randamu** | Libp2p enables direct peer-to-peer communication between Drand nodes, allowing for decentralized randomness generation and distribution. |
+| **Anoma** | By leveraging libp2p and introducing sovereign domains, Anoma has created a scalable infrastructure that can accommodate diverse node capabilities and network conditions. |
```

Comment:
Out of curiosity, where do they use libp2p in practice? Their network isn't live and its implemented in Elixir which has no libp2p.

I talked to Chris(CTO) several times convincing them to use libp2p, but they never decided to, unless something has changed recently.

Reply:
@Wondertan which company/project specifically? I know for a fact that Nethermind and Drand/Randmu do. I can't speak for Ceramic or Anoma.

Reply:
Oh, I meant Anoma. I know Ceramic has at least implicit dependency on libp2p, but I think its not the case for Anoma

Reply:
happy to remove to get this first draft merged ASAP and we can re-add after more research =]

Reply:
```suggestion
```